samples/sampler/stories/default/Drawing.js

```diff
@@ -8,11 +8,23 @@ export default {
 	title: 'Sandstone/Drawing'
 };
 
-export const _Drawing = (args) => (
-	<Drawing
-		disabled={args['disabled']}
-	/>
-);
+export const _Drawing = (args) => {
+	const disabled = args['disabled'];
+	return (
+		<section>
+			{disabled ? (
+				<p style={{fontSize: '70%', fontStyle: 'italic'}}>
+					<sup>*</sup>Drawing is not allowed while <code>disabled</code> is true.
+				</p>
+			) : (
+				<p />
+			)}
+			<Drawing
+				disabled={disabled}
+			/>
+		</section>
```

Comment:
```suggestion
		<section>
			<p style={{visibility: disabled ? 'visible' : 'hidden', fontSize: '70%', fontStyle: 'italic'}}>
				<sup>*</sup>Drawing is not allowed while <code>disabled</code> is true.
			</p>
			<Drawing
				disabled={disabled}
			/>
		</section>
```

Reply:
Adding an empty paragraph doesn't take the space for the text that appears when disabled is true. You can work with visibility.